scripts/fabricstat

```diff
@@ -177,11 +251,29 @@ class FabricQueueStat(FabricStat):
         asic_name = '0'
         if self.namespace:
             asic_name = multi_asic.get_asic_id_from_name(self.namespace)
+
+        cnstat_fqn_file_queue_name = cnstat_fqn_file_queue + asic_name
+        cnstat_cached_dict={}
+        if os.path.isfile(cnstat_fqn_file_queue_name):
+            try:
+                cnstat_cached_dict = json.load(open(cnstat_fqn_file_queue_name, 'r'))
+            except IOError as e:
```

Comment:
>except [](http://example.com/codeflow?start=12&length=6)

do you want to return earlier if exception caught?

Reply:
If the file not exist, I think we can still print out the counters ( with no cleaned ) , instead of print out nothing .
I read the clean code for other counters ( not exactly the same code piece as what I wrote ), but when they don't have the cached file, it still print out the uncleared counters. Please suggest if this behavior is ok or you want to print out nothing in this case. thank you